### PR TITLE
Atom rendering Youtube

### PR DESCRIPTION
**Pull request by liywjl**
## What does this change?
Update atoms-rendering to enable youtube analytics.

## Why?
Need to add analytics to youtube

### CODE REVIEW

**Review by oliverlloyd — CHANGES_REQUESTED** [review]
Can we add some tests for this atom? We've got the `atom.interactivity.spec.js` where we already check other atoms and because we removed the tests on atoms-rendering we currently have reduce coverage. I know there's pressure to get this one out but I think now is the right time to reintroduce this coverage in DCR to make sure that this functionality doesn't break in production without us knowing about it

**Comment by liywjl** [comment]
superseded by https://github.com/guardian/dotcom-rendering/pull/2199

File: package.json [hunk 1]
```diff
@@ -111,7 +111,8 @@
         "pretty-bytes": "^4.0.2",
         "reset-css": "^5.0.1",
         "sanitize-html": "^2.1.2",
-        "web-vitals": "^0.2.4"
+        "web-vitals": "^0.2.4",
+        "youtube-player": "^5.5.2"
```
Comment (oliverlloyd) [inline package.json:115]: Shouldn't this be a dependency of the atoms-rendering package?

**Comment by liywjl** [comment]
superceded by https://github.com/guardian/dotcom-rendering/pull/2285

**Pull request closed without merging**